app/helpers/miq_ae_class_helper.rb

```diff
@@ -41,7 +44,7 @@ def class_prefix(cls)
     case cls.to_s.split("::").last
     when "MiqAeClass"
       "aec"
-    when "MiqAeNamespace"
+    when "MiqAeDomain", "MiqAeNamespace"
```

Comment:
@jrafanie this method is used by views to add prefix on the links in the list on the right side.